app/src/main/res/values/styles.xml

```diff
@@ -1,11 +1,10 @@
 <resources>
 
     <!-- Base application theme. -->
-    <style name="AppTheme" parent="Theme.AppCompat.Light.DarkActionBar">
+    <style name="AppTheme" parent="Theme.AppCompat.Light.NoActionBar">
```

Comment:
Can we please use "Theme.MaterialComponents.Light.NoActionBar" instead? I need that to use the chips in my branch